napari_builtins/builtins.yaml

```diff
@@ -120,6 +120,12 @@ contributions:
     - id: napari.data.text
       title: Generate text sample
       python_name: napari_builtins._skimage_data:text
+    - id: napari.data.balls_2d
```

Comment:
Based on existing Binary Blobs, these should be alphabetical after Astronaut as:
- Balls
- Balls (3D)

Reply:
Done